advanced-creation.md

```diff
@@ -114,3 +112,31 @@ const unicorn = got.create(settings);
 // Same as:
 const unicorn = got.extend({headers: {unicorn: 'rainbow'}});
 ```
+
+#### got.forward(instance)
```

Comment:
I think `forward` is a confusing name as it sounds like you're forwarding the request. If this were to be added, it would make more sense to just let `got.extend` and `got.create` accept a parent instance.

Reply:
What if you use two different libs (instances of `got`)? Then you have no possibility to do that. Wait, you could do that...

Reply:
`got.extend(instance)` people would think it'll only merge options (not handlers), the same as `got.extend(instance.defaults.options)` but no the same as `got.forward(instance)`. I'll rename it to `got.merge`.